### PR TITLE
create intro_git app

### PR DESCRIPTION
**Pull request by hofanguyenn**

### CODE REVIEW

File: intro_git/models.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from django.db import models
+
+# Create your models here.
```
Comment (KickSone) [inline intro_git/models.py:3]: Create a Intro model here.

**Review by KickSone — COMMENTED** [review]
Change that.

File: intro_git/models.py (hunk 1 above)
Comment (KickSone) [inline intro_git/models.py:3]: No, add a model for it.

**Review by KickSone — COMMENTED** [review]
Let change that.

**Comment by hofanguyenn** [comment]
Sorry, I haven't searched fault this pull request

**Pull request closed without merging**